src/Converter.php

```diff
@@ -55,10 +55,11 @@ class Converter
      *
      * @param Project $project
      * @param File    $file
+     * @param bool    $nullableTypes
      *
      * @return string
      */
-    public function convert(Project $project, File $file): string
+    public function convert(Project $project, File $file, bool $nullableTypes): string
```

Comment:
Is the new required argument a breaking change? (e.g. for code using the converter as a library)

Would a bit field be better than a boolean flag? (e.g. to add more options in the future)

Reply:
We're still in 0.x versions, BC breaks are OK according to semver.

Reply:
OK thanks.